tests/scenario/test_machine_charm/test_alert_labels.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import json
+
+import machine_charm
+from scenario import Context, PeerRelation, Relation, State, SubordinateRelation
+
+from tests.scenario.helpers import get_charm_meta
+
+
+def test_metrics_alert_rule_labels(vroot):
```

Comment:
Could you add a docstring for the purpose of the test?

Reply:
Done.